<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QListWidget" name="outputWidget"/>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item>
     <widget class="QScrollArea" name="inputWidget">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="contents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>212</width>
         <height>519</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
